.pre-commit-config.yaml

```diff
@@ -52,12 +50,4 @@ repos:
   - repo: https://github.com/asottile/yesqa
     rev: v1.4.0
     hooks:
-      -   id: yesqa
-  - repo: https://github.com/asottile/add-trailing-comma
```

Comment:
Removing this since it is handled by `black`.

Reply:
Not entirely. Black does not add trailing commas when the arguments include *, *args or **kwargs.

I vote we keep add-trailing-comma

Reply:
Ok, we can add it in a follow-up PR